apps/web/src/pages/profile/penalties.tsx

```diff
@@ -14,4 +80,93 @@ PenaltiesPage.getLayout = (page) => {
   )
 }
 
+interface AccordionSetup {
+  title: string
+  category: string
+  givenAt: Date
+  duration: number
+  details: string
+}
+
+const PenaltyAccordion = (props: AccordionSetup) => {
+  const givenAtDate =
+    props.givenAt.getDate().toString() +
+    "." +
+    props.givenAt.getMonth().toString() +
+    "." +
+    props.givenAt.getFullYear().toString()
+  const expirationDate = new Date(props.givenAt.getTime() + props.duration)
+  const expirationDateFormat =
+    expirationDate.getDate().toString() +
+    "." +
+    expirationDate.getMonth().toString() +
+    "." +
+    expirationDate.getFullYear().toString()
+  return (
+    <Accordion type="single" collapsible>
+      <AccordionItem value="item-1">
+        <AccordionTrigger className="accordions ml-1 text-lg">
+          <span className="flex">
+            <DotFilledIcon className="mt-1" />
+            {props.title}
+          </span>
+        </AccordionTrigger>
+        <AccordionContent className="ml-4">
+          <div className="flex flex-col space-y-4">
+            <p>
+              {" "}
+              Du har fått en prikk på grunn av {props.details} den {givenAtDate}
+            </p>
+            <p className="text-lg">
+              <span className="font-bold">Katergori: </span>
+              {props.category}
+            </p>
+            <p className="text-lg">
+              <span className="font-bold">Utløpsdato: </span>
+              {expirationDateFormat}
+            </p>
+          </div>
+        </AccordionContent>
+      </AccordionItem>
+    </Accordion>
+  )
+}
+
+const activePenalties = listOfDummyMarks.length
```

Comment:
probably look into retrieving this from an api

Reply:
No api to use yet. Implement later